Makefile

```diff
@@ -30,8 +30,8 @@ cleanup:
 	- docker rm -f eventstore_local_noauth
 
 eventstore_docker:
-	docker run -d --name eventstore_local -p 2113:2113 -p 1113:1113 eventstore/eventstore
-	docker run -d --name eventstore_local_noauth -p 22113:2113 -p 11113:1113 eventstore/eventstore
+	docker run -d --name eventstore_local -p 2113:2113 -p 1113:1113 eventstore/eventstore:release-5.0.8
+	docker run -d --name eventstore_local_noauth -p 22113:2113 -p 11113:1113 eventstore/eventstore:release-5.0.8
```

Comment:
Leaving the tag out altogether or using `latest` does not seem to bring the latest image. At the time of this PR, without this, the tests could not be executed.
![image](https://user-images.githubusercontent.com/26380467/84396553-0c570700-ac07-11ea-9b06-fe6e02830051.png)

Below the difference between eventstore container without and with a tag:
![image](https://user-images.githubusercontent.com/26380467/84397376-54762980-ac07-11ea-9da2-a24627c7fc7b.png)

![image](https://user-images.githubusercontent.com/26380467/84397481-60fa8200-ac07-11ea-9bf8-d588dc1a524a.png)

Reply:
could this be because you already had a version of eventstore locally?  specifying no tag, or specifying `:latest` has the same effect, it just uses your local cached image if you have one.

Reply:
Don't believe that to be the case because the issue occurred in Travis and only afterwards tried locally and saw the same problem (the first screenshot is from Travis).

There was also this [issue](https://github.com/EventStore/eventstore-docker/issues/66 ) recorded that was supposedly fixed 3 days ago, however after reverting these changes in our code the [build](https://travis-ci.org/github/madedotcom/photon-pump/builds/699593993) failed with the same error.